src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.x
+---
+
+ * Added bright colors to `Color` class.
```

Comment:
Should be moved to 5.3 section and and:
```suggestion
 * Add support for bright colors
```

Reply:
It will be useful if you can specify the colors added.

Reply:
> It will be useful if you can specify the colors added.

not really sure on my side